ui/src/app/applications/components/utils.tsx

```diff
@@ -397,6 +397,37 @@ export const deletePopup = async (ctx: ContextApis, resource: ResourceTreeNode,
     );
 };
 
+function getResourceActionsMenuItems(resource: ResourceTreeNode, application: appModels.Application, appContext: AppContext): Observable<ActionMenuItem[]> {
```

Comment:
Perhaps it could make sense for the `application` parameter to instead be `metadata`, since that's the only part of `application` that I can see being used within this function?